site2/website/release-notes.md

```diff
@@ -1,6 +1,84 @@
 
 ## Apache Pulsar Release Notes
 
+### 2.6.3 &mdash; 2020-11-25 <a id=“2.7.0”></a>
+
+#### Broker
+
+- Update BookKeeper version to 4.11.1 [#8604](https://github.com/apache/pulsar/pull/8604)
+- Use correct configuration for zk-cache expire time [#8302](https://github.com/apache/pulsar/pull/8302)
+- Refresh ZooKeeper-data cache in background to avoid deadlock and blocking IO on ZK thread [#8304](https://github.com/apache/pulsar/pull/8304)
```

Comment:
```suggestion
- Refresh ZooKeeper-data cache in background to avoid deadlock and blocking IO on the ZooKeeper thread [#8304](https://github.com/apache/pulsar/pull/8304)
```